modules/library/http/src/main/java/org/geotools/http/HTTPClient.java

```diff
@@ -133,6 +133,12 @@ default HTTPResponse get(URL url, Map<String, String> headers) throws IOExceptio
     /** @param password the HTTP BASIC Authentication password */
     public void setPassword(String password);
 
+    /** @return the optional querystring ExtraParams, or {@code null} if not set */
```

Comment:
```suggestion
    /** Optional query parameters.
      * @return querystring ExtraParams, or {@code null} if not set
      */
```

We tend to prefer empty objects, for example an empty map, rather than null as per GeoTools style guide.

Reference "do not return null" https://docs.geotools.org/latest/developer/conventions/code/null.html